tools/audio/extraction/config.py

```diff
@@ -0,0 +1,214 @@
+# config.py
```

Comment:
OoT just recently add a YAML style config I think? Would it be better if that was used instead of audio having its own version? Would add some overhead though.